runtime/command/run/run.go

```diff
@@ -49,56 +52,95 @@ func (c *Command) compileAndStart(ctx context.Context, ln net.Listener) (*exe.Cm
 	return process, nil
 }
 
+func replacePreviousLine(msg string) {
+	// Move cursor up and delete line based on ANSI Escape Sequences: https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797
+	switch runtime.GOOS {
+	case "windows":
+		// TODO: Test this on Windows, don't know will it work or not
+		fmt.Print("\033[1A\033[0K")
+	case "linux":
+		fmt.Print("\033[1A\033[0K")
+	default:
+		console.Error("Can't detect your operating system")
+	}
```

Comment:
Let's remove this check.

`fmt.Print("\033[1A\033[0K")` seems to work on OSX and I guess it works for you on Linux. 

Windows experts can help us make this better, when they give Bud a go.

Reply:
If helpful I found [another implementation for a spinner](https://github.com/briandowns/spinner/blob/master/spinner.go#L412-L431), which is used within the [Github's CLI](https://github.com/cli/cli/blob/7a3d02df470252be0dbf8c7bd44176bcd48ba4ef/pkg/iostreams/iostreams.go#L262).

It seems like this one is slightly more battle-tested 😆